config/initializers/yabeda.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+return if Settings.scc.try(:metrics) && (Settings.scc.metrics.try(:enabled) && (!!Settings.scc.metrics.enabled  == false))
```

Comment:
Maybe `Settings` supports `Hash#dig` ?

It would make it way cleaner to call it like:

```ruby
enabled = Settings.dig(:scc, :metrics, :enabled)
```